src/v0/destinations/google_adwords_enhanced_conversions/networkHandler.js

```diff
@@ -48,26 +48,32 @@ const getConversionActionId = async (method, headers, params) => {
       headers,
       method,
     };
-    const response = await httpSend(requestBody);
-    if (!response.success && !isHttpStatusSuccess(response.response?.response?.status)) {
+    const { processedResponse: gaecConversionActionIdResponse } = await handleHttpRequest(
+      'constructor',
+      requestBody,
+    );
+    if (!isHttpStatusSuccess(gaecConversionActionIdResponse.status)) {
       throw new NetworkError(
         `"${get(
-          response,
-          'response.response.data[0].error.message',
+          gaecConversionActionIdResponse,
+          'response[0].error.message',
```

Comment:
What if `gaecConversionActionIdResponse` don't have `response[0].error.message`? It will return an empty string again. Can we verify that beforehand if we didn't get the response in the expected format we will share the whole response in the error message?

Reply:
Updated it. Returning the stringified response in case it is not in expected format.